apps/cowswap-frontend/src/modules/swap/containers/SwapWidget/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { useCallback, useMemo, useState } from 'react'
+import React, { useCallback, useMemo, useState } from 'react'
```

Comment:
Why import `React`?

Reply:
IDEA does it :( I don't know how to prevent it.
Anyway, deleted it